src/sasctl/utils/functionInspector.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# Copyright © 2022, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import sys
+
+
+def findPackageBytecodes(inst):
+    from types import CodeType, FunctionType, ModuleType
```

Comment:
move all the imports to top of file (they're standard packages that are always available and are always going to end up being loaded anyways).